CHANGES.rst

```diff
@@ -5,7 +5,8 @@ Changelog
 2.5.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Fix condition from last fix :(
```

Comment:
@sneridagh can we be a bit more specific here please? :)

Reply:
done